src/lib/agent/Agent.ts

```diff
@@ -98,12 +103,20 @@ export class Agent {
     const { publicDid, publicDidSeed } = this.agentConfig;
     if (publicDid && publicDidSeed) {
       // If an agent has publicDid it will be used as routing key.
-      this.wallet.initPublicDid(publicDid, publicDidSeed);
+      this.wallet.initAgentDid(publicDid, publicDidSeed);
     }
```

Comment:
I just noticed that `initAgentDid` is using `publicDid` from the config. Why is it using `publicDid` and not `agentDid` or something?

Reply:
I gave it another look and you're right we can use just `publicDid` 👍